notebooks/vertex_genai/solutions/prompt_management_and_context_caching_with_gemini.ipynb

```diff
@@ -0,0 +1,887 @@
+{
```

Comment:
<p>Could you change the GCS paths in the following way?</p><p> </p>

<pre class="ql-syntax" spellcheck="false">gs://cloud-samples-data/generative-ai/pdf/2312.11805v3.pdf
-&gt;
gs://asl-public/data/generative-ai/pdf/2312.11805v3.pdf

gs://cloud-samples-data/generative-ai/pdf/2403.05530.pdf"
-&gt;
gs://asl-public/data/generative-ai/pdf/2403.05530.pdf
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>